docs/07-how_to_run_a_node/03-run_using_docker/02-run-rpc-node.md

```diff
@@ -0,0 +1,178 @@
+
+```bash
+docker logs nh-node-simple_node --tail 100
+```
+
+and you'll see there is no more that long sequence of logs reporting initial download of blockchain data (`Starting new tree with id: XXX`).
+
+If for any reason you need to fully erase blockchain data you can do this by stopping the container and deleting the Docker volume:
+
+```bash
+docker compose -f nh-node-docker-compose.yaml down
+docker volume rm zkverify-rpc_node-data
+```
+
+It's now time to interact a bit with your node, and the best tool for achieving this is PolkadotJS. Open up your preferred browser and type in the search bar this [URL](https://polkadot.js.org/apps/#/explorer). Then make sure you are targeting the local node by checking through the dropdown panel in the top left corner:
+
+![alt_text](./img/polkadotjs_check_network.png)
+
+*In case you are unable to connect, check manually that inside `.env` file the value for configuration `NH_CONF_RPC_EXTERNAL` is set to `true` and that the value for configuration `NODE_NET_RPC_WS_PORT` is matching the one you eventually typed in the `custom endpoint` textbox.*
+
+In the first place, you can try submitting some RPC commands for querying the blockchain state. An example could be asking for the block hash at a specific height and then using the result to ask for the full block body.
+
+Navigate to section `Developer` then subsection `RPC calls` and select `chain` endpoint, command `getBlockHash`; make sure to enable `include option` flag, then type in the block number (100 in this example), finally click on `Submit RPC call` button.
+
+![alt_text](./img/polkadotjs_query_block_hash.png)
+
+What you get as a response is indeed the block hash (`0xa048…ac51` here); with this result you can submit a second command for getting the entire body of the block. In an analogous way, select command `getBlock` and paste in as input the hash you have just received.
+
+Again, the node is responding to your request, this time providing the full body of the queried block.
+
+![alt_text](./img/polkadotjs_query_block.png)
+
+Note that if during the execution of script `init.sh` you selected non-archival node you could incur in an error like `-32000: Client error: Api called for an unknown Block: State already discarded for 0xa048…ac51`, reporting too old data have been pruned.
+
+Another type of operation you can perform is sending some tokens from an account to another, clearly is essential you possess some tokens on one of yours. In Substrate accounts are associated to a pair of keys, one private and one public (very similar to Bitcoin addresses or Ethereum EOAs).
+
+In order to sumbit extrinsics (transactions, in Substrate terminology) with PolkadotJS from one of your account, you need to import it within the application.
+
+Navigate to section `Settings` then subsection `account options`, open drop-down list `in-browser account creation`, select option `Allow local in-browser account storage` and click on `Save` button:
```

Comment:
Is enabling in-browser account storage best practice? See this note from polkadot.js:
```
It is recommended that you create/store your accounts securely and externally from the app. On Chrome the following browser extensions are available for use -

[polkadot-js extension](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd) (Basic account injection and signer)
Accounts injected from any of these extensions will appear in this application and be available for use. The above list is updated as more extensions with external signing capability become available. [Learn more...](https://github.com/polkadot-js/extension)

Since some extensions, such as the polkadot-js extension, protects you against all community reported phishing sites, there are valid reasons to use them for additional protection, even if you are not storing accounts in it.
```

Reply:
I get your point, but at the moment since we are in testnet (so no real value for the ACME token) and the guide is already quite long to go through I prefer not to bring into an additional tool which would make the bootstrap less quick (already a lot of requirements and tools are needed).
For a future milestone we can think about consolidating this part.